src/Raven.Server/Documents/Handlers/Processors/AbstractStatsHandlerProcessorForGetStudioFooterStats.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Studio;
+using Raven.Server.Web;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Processors
+{
+    internal abstract class AbstractStatsHandlerProcessorForGetStudioFooterStats<TRequestHandler, TOperationContext> : AbstractHandlerProcessor<TRequestHandler, TOperationContext>
+        where TRequestHandler : RequestHandler
+        where TOperationContext : JsonOperationContext
+    {
+        public AbstractStatsHandlerProcessorForGetStudioFooterStats([NotNull] TRequestHandler requestHandler, [NotNull] JsonContextPoolBase<TOperationContext> contextPool) : base(requestHandler, contextPool)
+        {
+        }
+
+        protected abstract ValueTask<FooterStatistics> GetFooterStatistics();
```

Comment:
Async methods need to have `Async` at the end

Reply:
changed